main.js

```diff
@@ -315,14 +212,89 @@ function createStatMMR(statistics, i) {
   return create$tr1
 }
 
-var searchPlayer = function (displayName, players) {
-  for (var i = 0; i < players.length; i++) {
-    if (displayName === players[i].displayName) {
-      return players[i]
+var searchPlayer = function (searchValue) {
+  var searchReq = new XMLHttpRequest()
+  searchReq.addEventListener('load', searchListener)
+  searchReq.open('GET', 'https://api.rocketleaguestats.com/v1/player?unique_id=' + searchValue + '&platform_id=1&apikey=UY5C423V91L8ZI1YEPXBBVP6Z8YVB5ME', true)
+  searchReq.responseType = 'json'
+  searchReq.send()
+  function searchListener() {
+    if (searchReq.status === 200) {
+      var statObject = this.response
+      generateStats(statObject)
     }
   }
 }
 
+var searchPlayerComparison = function (searchValue) {
+  var searchReq = new XMLHttpRequest()
+  searchReq.addEventListener('load', searchListener)
+  searchReq.open('GET', 'https://api.rocketleaguestats.com/v1/player?unique_id=' + searchValue + '&platform_id=1&apikey=UY5C423V91L8ZI1YEPXBBVP6Z8YVB5ME', true)
```

Comment:
Don't check API keys into Git! This secret is now public and can be used by anyone 😢